controller/classesController.js

```diff
@@ -0,0 +1,143 @@
+const classesService = require("../services/classesService");
+//console.log 삭제 필요
+const getClassesList = async(req,res) => {
+  try{
+    let { topCategoryName, subCategoryName, sortBy, search} = req.query;
+    console.log(req.query);
+    const result = await classesService.getClassesList(
+      topCategoryName,subCategoryName, sortBy, search
+    );
+    return res.status(200).json({ result });
+  } catch (err) {
+    console.log(err);
+    return res.status(500).json({ message: err.message });
+  };
+};
+
+const getUpcomingClasses = async(req,res) => {
+  try{
+    const result = await classesService.getUpcomingClasses();
+    return res.status(200).json({ message: result});
+
+  } catch (err) {
+    console.log(err);
+    return res.status(500).json({ message: err.message });
+  };
+
+};
+const getClassesDetail = async(req, res) => {
+  try {
+    const classId = req.params.classId;
+    const result = await classesService.getClassDetail(classId);
+    return res.status(200).json({ message: result });
+  } catch (err) {
+    console.error(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const getMyClassesList =  async(req,res) => {
+  try {
+    const userId = req.users.id;
+    const result = await classesService.getMyClassesList(userId);
+    return res.status(200).json({ message: result });
+  } catch (err) { 
+    console.error(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  };
+};
+
+const getHostClassesList =  async(req,res) => {
+  try {
+    const hostId = req.hosts.id;
+    const result = await classesService.getHostClassesList(hostId);
+    return res.status(200).json({ message: result });
+  } catch (err) {
+    console.error(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  };
+};
+const createClass = async(req,res) => {
+  try{
+    const hostId = req.hosts.id;
+    const result = await classesService.createClass(hostId,req);
+    return res.status(200).json({ result });
+  } catch(err) {
+    console.error(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  };
+};
+
+const updateClass = async(req,res) => {
+  try{
+    const hostId = req.hosts.id;
+    const classId = req.params.classid;
+    const result = await classesService.updateClass(hostId,classId,req);
```

Comment:
```suggestion
    const result = await classesService.updateClass(hostId, classId, ___);
   
```

1. 마찬가지로 req 객체 대신 필요한 다른 사항으로 수정해 주세요.
2. 인자들 사이에 띄어쓰기 해주세요..............ㅎ